query/tests/it/interpreters/interpreter_share_desc.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_base::base::tokio;
+use common_exception::Result;
+use databend_query::interpreters::*;
+use databend_query::sql::Planner;
+use futures::TryStreamExt;
+use pretty_assertions::assert_eq;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_desc_share_interpreter() -> Result<()> {
+    let ctx = crate::tests::create_query_context().await?;
+    let mut planner = Planner::new(ctx.clone());
+
+    // first create share;
+    {
+        let query = "create share t";
+        let (plan, _, _) = planner.plan_sql(query).await?;
+        let executor = InterpreterFactoryV2::get(ctx.clone(), &plan)?;
+        let stream = executor.execute().await?;
+        let _ = stream.try_collect::<Vec<_>>().await?;
+    }
+
+    // show create share
+    {
+        let query = "desc share t";
+        let (plan, _, _) = planner.plan_sql(query).await?;
+        let executor = InterpreterFactoryV2::get(ctx.clone(), &plan)?;
+        assert_eq!(executor.name(), "DescShareInterpreter");
+        let stream = executor.execute().await?;
+        let result = stream.try_collect::<Vec<_>>().await?;
+        let expected = vec![
+            "+------+------+-----------+",
```

Comment:
Why the results empty?

Reply:
> Why the results empty?

in https://databend.rs/doc/contributing/rfcs/share#describe-share there is `desc share` rfc:

```
Describes the shared objects that are included in a share.
```

cause column `grant_on` cannot be determined each time the test run, so just test an empty share without any shared objects.